svc-update/update/simpleUpdate.go

```diff
@@ -294,3 +303,52 @@ func sortTargetList(Targets []string) (map[string][]string, error) {
 	}
 	return returnList, nil
 }
+
+func expandAggregatesCollection(token string, links []string) ([]string, error) {
+	elements := make([]string, 0)
+	for _, link := range links {
+		if strings.Contains(link, "Aggregates") {
+			aggregate, err := getLinksFromAggregate(token, link)
+			if err != nil {
+				return nil, err
+			}
+			elements = append(elements, aggregate...)
+		} else {
+			elements = append(elements, link)
+		}
+	}
+	return elements, nil
+}
+
+func getLinksFromAggregate(token string, aggregate string) ([]string, error) {
+	aggregatorConn, err := services.ODIMService.Client(services.Aggregator)
+	if err != nil {
+		errMsg := fmt.Sprintf("Failed to create client connection with AggregateService: %v", err)
```

Comment:
Isn't it AggregationService?

Reply:
Confirming that it is AggregationService.

Reply:
Fixed